octo-app/src/app/_model/SystemUser.ts

```diff
@@ -4,9 +4,14 @@ export class SystemUser {
     password: string;
     firstName: string;
     lastName: string;
-    role: number;
+    role: {id: number, name: string};
     enabled: boolean;
     credentialsNonExpired: boolean;
     accountNonExpired: boolean;
     accountNonLocked: boolean;
+    boardUserJoins: {
```

Comment:
what is boardUserJoins doing here? I can't see why it's necessary and it should not be. TODO: delete this.